packages/graphql-key-transformer/src/KeyTransformer.ts

```diff
@@ -167,6 +196,11 @@ export class KeyTransformer extends Transformer {
           deleteResolver.Properties.RequestMappingTemplate,
         ]);
       }
+      if (syncResolver) {
```

Comment:
nit: duplicate code here maybe we can store the result of `this.isPrimaryKeyDirective(directive)` in a variable and pass that into `constructSyncResolver`?

Reply:
Its not duplicate. I am passing a istable boolean with value true/false depends on its for ddb table or GSI